### ISSUE TITLE
ReferenceError: WebSocket is not defined

### ISSUE THREAD

**Issue by binarykitchen** [maxogden/websocket-stream#92]
Happened few times with your latest version I checked out yesterday.

Points to this line

```
module.exports = WebSocket || MozWebSocket || window.WebSocket || window.MozWebSocket
```

User Agents were

Mozilla/5.0 (Linux; Android 6.0.1; Nexus 5X Build/MMB29P) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/41.0.2272.96 Mobile Safari/537.36 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)

and

Opera/9.80 (J2ME/MIDP; Opera Mini/4.2.23449/37.8474; U; en) Presto/2.12.423 Version/12.16

**Comment by mcollina** [comment]
Fixed and released as 3.2.1.

**Comment by binarykitchen** [comment]
thanks guys for the super-fast patch
